tests/MongoDB.Driver.Tests/Specifications/command-monitoring/CommandMonitoringTestRunner.cs

```diff
@@ -267,12 +312,6 @@ private BsonDocument MassageCommand(string commandName, BsonDocument command)
                     break;
                 case "update":
                     massagedCommand["ordered"] = massagedCommand.GetValue("ordered", true);
-                    foreach (BsonDocument update in (BsonArray)massagedCommand["updates"])
-                    {
-                        update["multi"] = update.GetValue("multi", false);
-                        update["upsert"] = update.GetValue("upsert", false);
```

Comment:
Can you explain why these lines were deleted?

Reply:
This is caused by these words in latest specification (https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst) for `upsert` and `multi` parameters:
> This option is sent only if the caller explicitly provides a value. The default is to not send a value.

The code above defaulted these parameters to `false` if they were not found, which broke new test cases (which expected the parameter wouldn't be present in response if it wasn't sent in the first place). That's why I removed it.